src/main/java/org/sagebionetworks/bridge/exporter/worker/TsvInfo.java

```diff
@@ -87,6 +90,18 @@ public int getLineCount() {
     }
 
     /**
+     * helper method to add a record id into the list
+     * @param recordId
+     */
+    public void addRecordId(String recordId) {
+        this.recordIds.add(recordId);
+    }
+
+    public List<String> getRecordIds() {
+        return this.recordIds;
+    }
```

Comment:
For safety, I would wrap this in an ImmutableList.copyOf(this.recordIds). This ensures that whoever is calling this doesn't end up accidentally modifying it.